src/leapfrogai_api/Makefile

```
@@ -7,6 +7,7 @@ export SUPABASE_ANON_KEY=$(shell supabase status | grep -oP '(?<=anon key: ).*')
 install:
 	python -m pip install ../../src/leapfrogai_sdk
 	python -m pip install -e .
+	python -m pip install ../../[dev]
```

As written, this doesn't work with all shell types.
```suggestion
	python -m pip install "../../.[dev]"
```

---

Good catch